libcloud/compute/drivers/vcloud.py

```diff
@@ -1044,8 +1063,6 @@ def ex_deploy_node(self, node):
                                       method='POST',
                                       headers=headers)
         self._wait_for_task_completion(res.object.get('href'))
-        res = self.connection.request(get_url_path(node.id))
```

Comment:
Ah, sorry, I was confused by the diff (I didn't fully expand it so I thought it's part of the public method).